source/Logger.js

```diff
@@ -0,0 +1,114 @@
+const endOfStream = require("end-of-stream");
+const uuid = require("uuid/v4");
+const { ITEM_TYPE, ITEM_TYPE_LOG_ENTRY } = require("./symbols.js");
+
+class Logger {
+    constructor(service) {
+        this.levels = {
+            LOGGER_ALERT: "alert",
+            LOGGER_ERROR: "error",
+            LOGGER_WARNING: "warning",
+            LOGGER_INFO: "info",
+            LOGGER_DEBUG: "debug"
+        };
+        this.entriesMax = 100;
+        this.service = service;
+        this.init();
+    }
+
+    init() {
```

Comment:
This could be called `initialise()` to match that of the `Scheduler` and others, and it could be called by `Service#initialise` instead of in the constructor.. :)

Reply:
Donezo 👍